js/package.json

```diff
@@ -52,11 +52,12 @@
     "wasm:go": "tinygo build -gc=leaking -no-debug -o main.wasm -target wasm ../cmd/wasm/main.go",
     "wasm:opt": "npx wasm-opt -O main.wasm -o main.wasm",
     "fixup": "echo '{\n  \"type\": \"commonjs\"\n}' > cjs/package.json",
+    "build:go-types": "cd ..; make types; cd js", 
```

Comment:
What do you think about this @joewagner? Is that too brittle?

Reply:
Seems good, Maybe do `&&` instead of `;`?